src/supervisor.tex

```diff
@@ -1092,6 +1092,14 @@ \subsection{Supervisor Memory-Management Fence Instruction}
       rs2}, except for entries containing global mappings.
 \end{itemize}
 
+If the value held in {\em rs1} is not a valid virtual address, then the
+SFENCE.VMA instruction has no effect.  No exception is raised in this case.
+
+\begin{commentary}
+This ensures it remains legal for implementations to simply ignore the
+value in {\em rs1}.
```

Comment:
But wouldn't an implementation that ignores the value in rs1 need to over-invalidate, not under-invalidate?  Performing no invalidation would certainly surprise software that's expecting at least the specified addresse to be invalidated.